dll/cpl/desk/lang/fr-FR.rc

```diff
@@ -59,17 +59,17 @@ CAPTION "Apparence"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "", IDC_APPEARANCE_PREVIEW, "PreviewWndClass", WS_VISIBLE | WS_BORDER, 7, 7, 232, 120
-    LTEXT "Style visuel", IDC_STATIC, 7, 130, 104, 9
+    LTEXT "Style visuel", IDC_STATIC, 7, 130, 104, 9//FIXME: add accel
```

Comment:
```suggestion
    LTEXT "&Style visuel", IDC_STATIC, 7, 130, 104, 9
```
(different wording than on Windows)

Reply:
I might do that, but please provide a 2k3 french screenshot first.

Reply:
Resolved by https://github.com/reactos/reactos/commit/c52631af1386e2965b623a08476d223b13f99330